README.md

```diff
@@ -198,6 +198,11 @@ brew install fd
 sudo port install fd
 ```
 
+… or with npm:
```

Comment:
I'd rather just add one section for `npm`-

Reply:
got it